src/platform/silabs/SiWx917/BLEManagerImpl.cpp

```diff
@@ -41,6 +41,7 @@ extern "C" {
 #include <rsi_driver.h>
 #include <rsi_utils.h>
 #include <stdbool.h>
+void M4_sleep_wakeup();
```

Comment:
The funciton declarion shouldn't go in the middle of the include section just because there is an `extern "C"`. After the includes, add another `extern "C"` and declare the function there.